forge-gui/res/cardsfolder/upcoming/eye_of_ojer_taq_apex_observatory.txt

```diff
@@ -0,0 +1,23 @@
+Name:Eye of Ojer Taq
+ManaCost:3
+Types:Artifact
+A:AB$ Mana | Cost$ T | Produced$ Any | SpellDescription$ Add one mana of any color.
+K:Craft:6 ExileCtrlOrGrave<2/Permanent.Other+withSharedCardType/permanent>:two that share a card type:the two
+AlternateMode:DoubleFaced
+DeckHints:Ability$Discard|Mill|Sacrifice
+DeckHas:Ability$Mill|Graveyard|Token
+Oracle:{T}: Add one mana of any color.\nCraft with two that share a card type {6} ({6}, Exile this artifact, Exile the two from among other permanents you control and/or cards from your graveyard: Return this card transformed under its owner's control. Craft only as a sorcery.)
+
+ALTERNATE
+
+Name:Apex Observatory
+ManaCost:no cost
+Types:Artifact
+K:ETBReplacement:Other:Tap
+SVar:Tap:DB$ Tap | Defined$ Self | ETB$ True | SubAbility$ DBChooseType | SpellDescription$ CARDNAME enters the battlefield tapped. As it enters, choose a card type shared among two exiled cards used to craft it.
+SVar:DBChooseType:DB$ ChooseType | Type$ Shared | TypesFromDefined$ ExiledWith | AILogic$ MostProminentComputerControlsOrOwns
```

Comment:
This currently doesn't work because ExiledWith is empty when it checks it

I need to fix it so it won't NPE which is not a big deal but I also need to figure out how to get at the Exiled cards... I can find them in the Replacing Objects -> Cause -> Paid Lists but that is going to be pretty gross to get at, I think?